wiki/redirect.yaml

```
@@ -62,16 +62,16 @@
 "packs":                       "Beatmaps/Packs"
 "pack":                        "Beatmaps/Packs"
 
-"best_of_2009":                "Best_Of/2009"
-"best_of_2010":                "Best_Of/2010"
-"best_of_2011":                "Best_Of/2011"
-"best_of_2012":                "Best_Of/2012"
-"best_of_2013":                "Best_Of/2013"
-"best_of_2014":                "Best_Of/2014"
-"best_of_2015":                "Best_Of/2015"
-"best_of_2016":                "Best_Of/2016"
-
 "contest":                     "Contests"
+"best_of":                     "Contests#best-of"
+"best_of_2009":                "Contests#best-of"
```

could link to a more specific anchor
```suggestion
"best_of_2009":                "Contests#2009"
```
same for 2010-2013